### PR TITLE
chore: Minor improvements to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by cveticm**
## Proposed changes

* Change to debug configuration example from `atlas login` to `atlas dbuser create [...]` to show how to deal with multiple args and flags.
* Rewords environment setup section for clarity and readability.
* Adds minor grammar corrections.

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

## Further comments

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,36 +88,41 @@ Please add the following line to your settings.json file :
...
             "env": {},
             "args": [
-              "login"
+              "dbuser",
+              "create",
+              "atlasAdmin", 
+              "--username",
+              "myUser",
+              "--password",
+              "123abc"
```
Comment (cveticm) [inline CONTRIBUTING.md:121]: I suggest changing the example configuration to something with more args and flags required to more clearly show how the configuration needs to be altered for alternate commands.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,15 +55,16 @@ To make the contribution process as seamless as possible, we ask for the followi
 #### Environment
 
 - Fork the repository.
-- Clone your forked repository locally.
-- We use Go Modules to manage dependencies, so you can develop outside your `$GOPATH`.
-- We use [golangci-lint](https://github.com/golangci/golangci-lint) to lint our code, you can install it locally via `make setup`.
+- Clone your fork locally.
+- Set up development environment:
```
Comment (corryroot) [inline CONTRIBUTING.md:59]:
```suggestion
- Set up your development environment:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -55,15 +55,16 @@ To make the contribution process as seamless as possible, we ask for the followi
...
+  - We use Go Modules to manage dependencies, so you can develop outside your `$GOPATH`.
+  - Run `make setup` to install required dependencies and developer tools. This includes [golangci-lint](https://github.com/golangci/golangci-lint), which we use to lint our code.
```
Comment (corryroot) [inline CONTRIBUTING.md:61]:
```suggestion
  - Run `make setup` to install required dependencies and developer tools including [golangci-lint](https://github.com/golangci/golangci-lint), which we use to lint our code.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -87,36 +88,41 @@ Please add the following line to your settings.json file :
...
 
-This will enable compilation for unit test and end-to-end tests.
+This will enable compilation for unit and end-to-end tests.
 
 #### Debugging in VSCode
 
-To debut in VSCode you need to create a debug configuration for the command with required arguments.
-Run the following commands to 
+To debug in VSCode, you need to create a debug configuration for the command with required arguments.
+Run the following commands to create a new launch.json file for the debugger:
```
Comment (corryroot) [inline CONTRIBUTING.md:96]:
```suggestion
To debug in VSCode, you must create a debug configuration for the command with the required arguments.
Run the following commands to create a new launch.json file for the debugger:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -87,36 +88,41 @@ Please add the following line to your settings.json file :
...
 
 ```
 touch .vscode/launch.json
 ```
-Then put the following configuration into the file.
-Review and replace command name and arguments depending on the command you are using.
+Then put the following example configuration into the file.
+Review and replace the command name and arguments depending on the command you wish to debug.
```
Comment (corryroot) [inline CONTRIBUTING.md:102]:
```suggestion
Then put the following example configuration into the file.
Review and replace the command name and arguments depending on the command you want to debug.
```

**Review by corryroot — APPROVED** [review]
LGTM! I commented with some copy suggestions.

File: CONTRIBUTING.md (hunk 3 above)
Comment (blva) [inline CONTRIBUTING.md:61]: should we also mention any of those two commands:
```
deps                           Download go module dependencies
devtools                       Install dev tools
```

Comment (cveticm) [reply CONTRIBUTING.md:61]: Looking into the makefile, `make setup` already runs deps, devtools and link-git-hooks so I don't think it is needed.

Comment (blva) [reply CONTRIBUTING.md:61]: ah, perfect! thanks

**Review by blva — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (cveticm) [reply CONTRIBUTING.md]: Added 👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (cveticm) [reply CONTRIBUTING.md]: Added

File: CONTRIBUTING.md (hunk 4 above)
Comment (cveticm) [reply CONTRIBUTING.md]: Added

File: CONTRIBUTING.md (hunk 5 above)
Comment (cveticm) [reply CONTRIBUTING.md]: Added

**Review by blva — APPROVED** [review]

**Pull request merged**